getparty.go

```diff
@@ -332,8 +333,16 @@ func (cmd Cmd) follow(jar http.CookieJar, userUrl string) (session *Session, err
 			jar.SetCookies(u, cookies)
 		}
 	}
-	client := cleanhttp.DefaultClient()
-	client.Jar = jar
+
+	transport, err := cmd.getTransport()
```

Comment:
follow method should pass `pooled = false` to `cmd.getTransport()`, see my comment below.